atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -12,11 +12,9 @@ Atlas supports Node.js applications using popular frameworks like [Faust.js](htt
 
 ### 1. Prepare your app for deployment
 
-First make sure that you've pushed your application's code to your GitHub repository. If you're not familiar with pushing code to a GitHub repository, see this guide: [Adding an existing project to GitHub using the command line](https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-an-existing-project-to-github-using-the-command-line).
+When building the front-end app, WPEngine will use ```npm run build``` by default. If you want to specify a build command, you can do so in one of two ways:
```

Comment:
@dgallardox "WP Engine" should be two words-- need to add a space between them.
Also, \```npm run build\``` can be simplified to \`npm run build\`. Only one set of backticks is necessary for inline code.